base/sparse/sparsevector.jl

```diff
@@ -919,7 +919,8 @@ hvcat{T}(rows::Tuple{Vararg{Int}}, xs::_TypedDenseConcatGroup{T}...) = Base.type
 
 # zero-preserving functions (z->z, nz->nz)
 broadcast(::typeof(abs), x::AbstractSparseVector) = SparseVector(length(x), copy(nonzeroinds(x)), abs.(nonzeros(x)))
-for op in [:abs2, :conj]
+broadcast(::typeof(abs2), x::AbstractSparseVector) = SparseVector(length(x), copy(nonzeroinds(x)), abs2.(nonzeros(x)))
+for op in [:conj]
```

Comment:
IIUC `conj` methods for vectors/matrices should persist per discussion in https://github.com/JuliaLang/julia/pull/18495#issuecomment-267215901. Thoughts? Thanks!